### PR TITLE
Fix: jenkinsfile not take standard java syntax

### PR DESCRIPTION
**Pull request by zdtsw**
see error from installer job:
`Exception in buildAndTest: groovy.lang.MissingPropertyException: No such property: buildCli for class: WorkflowScript`

### CODE REVIEW

**Comment by zdtsw** [comment]
maybe can try and test it to see the error is gone from "installer" job? 
since GH action does not really run on this change.

**Review by steelhead31 — APPROVED** [review]
LGTM

File: linux/Jenkinsfile [hunk 1]
```diff
@@ -245,7 +245,7 @@ def buildAndTest(String DISTRO, String buildArch) {
     }
 }
 
-private void buildCLi(String DISTRO, String buildArch, String GPG_KEY = null) {
+def buildCLi(String DISTRO, String buildArch, String GPG_KEY = null) {
     def buildCLI = "./gradlew package${TYPE}${DISTRO} check${TYPE}${DISTRO} --parallel -PPRODUCT=${env.PRODUCT} -PPRODUCT_VERSION=${VERSION} -PARCH=${buildArch}"
     if (GPG_KEY) {
         buildCli += " -PGPG_KEY=${GPG_KEY}"
```
Comment (andrew-m-leonard) [inline linux/Jenkinsfile:251]: Wrong variable case here: buildCli should be buildCLi

Comment (zdtsw) [reply linux/Jenkinsfile:251]: thats on Gerd, but i will fix it for him

**Review by andrew-m-leonard — APPROVED** [review]

File: linux/Jenkinsfile (hunk 1 above)
Comment (zdtsw) [reply linux/Jenkinsfile]: @steelhead31 is testing it, and if it is good we can merge this PR first and get JRE publish asap.
After that you are free to do more polish work @ascheman , how you think?

Comment (ascheman) [reply linux/Jenkinsfile]: I suggest to use #605 (just let me do a local test on my private Jenkins first).

Comment (ascheman) [reply linux/Jenkinsfile]: Or @steelhead31 could test my PR already including GPG signing (but without publishing the binaries of course)?

**Comment by ascheman** [comment]
#605 seems to work better, I suggest to close this one and to approve #605 instead, @zdtsw / @steelhead31 / @andrew-m-leonard

**Comment by zdtsw** [comment]
> #605 seems to work better, I suggest to close this one and to approve #605 instead, @zdtsw / @steelhead31 / @andrew-m-leonard

can @steelhead31 test Gerd's #605 ? if all happy with result, i can close this PR

**Comment by zdtsw** [comment]
close of mine, in favor of #605

**Pull request closed without merging**